remoteworkitem/jira_test.go

```diff
@@ -27,3 +30,43 @@ func TestJiraFetch(t *testing.T) {
 	}
 
 }
+
+func TestJiraFetchWithRecording(t *testing.T) {
+	resource.Require(t, resource.UnitTest)
+	r, err := recorder.New("../test/data/jira_fetch_test")
+	if err != nil {
+		t.Error(err)
+	}
+	defer r.Stop()
+
+	h := &http.Client{
+		Timeout:   100 * time.Second,
+		Transport: r.Transport,
+	}
+
+	f := jiraIssueFetcher{}
+	j := JiraTracker{URL: "https://issues.jboss.org", Query: "project = Arquillian AND status = Closed AND assignee = aslak AND fixVersion = 1.1.11.Final"}
```

Comment:
Maybe add an "order" in the query to ensure the order check below stays the same